### PR TITLE
docs: short guide for get()

### PR DESCRIPTION
**Pull request by JohannesDienst-askui**
Add a short guide on how to use `get()` to extract information out of the UI.

### CODE REVIEW

File: docs/docs/general/03-Guides/extract-information.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+
+## Extract the Value
+You can extract the value by reading the `text`-property of the `DetectedElement`.
+
+**Notice: `textFieldText` is an array. You have to iterate over it or use an index to access the `DetectedElement`.**
+
+```typescript
+// Access by index
+// textFieldTextValue gets the value 'Search'
+const textFieldTextValue = textfieldText[0].text;
```
Comment (adi-wan-askui) [inline docs/docs/general/03-Guides/extract-information.md:62]: You should make sure that testfieldText[0] is actually set (and correctly cased ;P), i.e., the array is not empty, shouldn't you? E.g.,
```typescript
const textFieldTextValue = textFieldText[0]?.text
```
or even
```typescript
const textFieldTextValue = textFieldText?.[0]?.text
```

**Review by adi-wan-askui — CHANGES_REQUESTED** [review]
Nice :)

File: docs/docs/general/03-Guides/extract-information.md (hunk 1 above)
Comment (JohannesDienst-askui) [reply docs/docs/general/03-Guides/extract-information.md:62]: What a nasty copy&paste error from me 😢  Thanks for catching it!

Also will include the suggestion with `?`

In my opinion the second `?` is not needed as `text` is always defined in this case. Any objections?

File: docs/docs/general/03-Guides/extract-information.md [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+
+
+
+```text
+[
+  DetectedElement {
+    name: 'text',
+    text: 'Search',
+    bndbox: BoundingBox { xmin: 488, ymin: 138, xmax: 548, ymax: 164 },
+    colors: [ '', '', '' ]
```
[the remark this answers is not in the document]
Comment (JohannesDienst-askui) [reply docs/docs/general/03-Guides/extract-information.md:49]: In version `0.10.1` the output is still like this. How will it change @mlikasam-askui ?

File: docs/docs/general/03-Guides/extract-information.md [hunk 3]
```diff
@@ -59,7 +59,7 @@ You can extract the value by reading the `text`-property of the `DetectedElement
 ```typescript
 // Access by index
 // textFieldTextValue gets the value 'Search'
-const textFieldTextValue = textfieldText[0].text;
+const textFieldTextValue = textFieldText?[0]?.text;
```
Comment (adi-wan-askui) [inline docs/docs/general/03-Guides/extract-information.md:62]: Is this right? I think it is:
```suggestion
const textFieldTextValue = textFieldText?.[0]?.text;
```

**Review by adi-wan-askui — APPROVED** [review]
Feel free to resolve the remarks and merge :)

**Pull request merged**